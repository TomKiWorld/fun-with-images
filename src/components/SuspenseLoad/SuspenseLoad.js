import React, { Suspense } from 'react';
import Preloader from '../Preloader/Preloader';

const SuspenseLoad = (props) => {
  return (
    <Suspense fallback={<Preloader />}>
      {props.children}
    </Suspense>
  );
}

export default SuspenseLoad;
