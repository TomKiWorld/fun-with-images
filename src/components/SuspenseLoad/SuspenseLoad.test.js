import { shallow } from 'enzyme';
import React from 'react';
import SuspenseLoad from './SuspenseLoad';

describe('SuspenseLoad Component', () => {
  const child = () => <h1>Test</h1>;
  it('Expect to render SuspenseLoad Component', () => {
    expect(shallow(<SuspenseLoad child/>)).toMatchSnapshot();
  });
});
