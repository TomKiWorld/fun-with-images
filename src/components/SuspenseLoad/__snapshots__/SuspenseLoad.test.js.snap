// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuspenseLoad Component Expect to render SuspenseLoad Component 1`] = `ShallowWrapper {}`;
